frontend/src/views/CellGuide/components/CellGuideCard/components/MarkerGeneTables/hooks/computational_markers.ts

```diff
@@ -11,95 +15,129 @@ interface ComputationalMarkerGeneTableData {
   pc: string;
 }
 
-function _getSortedOrganisms(
-  genes: ComputationalMarkersQueryResponse
-): string[] {
-  const organisms = new Set<string>();
-  for (const markerGene of genes) {
-    if (markerGene.marker_score < FMG_GENE_STRENGTH_THRESHOLD) continue;
-    organisms.add(markerGene.groupby_dims.organism_ontology_term_label);
-  }
-  return Array.from(organisms).sort((a, b) => {
-    if (a === HOMO_SAPIENS) return -1;
-    if (b === HOMO_SAPIENS) return 1;
-    return a.localeCompare(b);
-  });
+enum selectionCriteriaErrorCode {
```

Comment:
Do we need TissueMismatch error code in canonical marker genes? First hunch is that we do not, since canonical marker genes only ever has one "No marker gene" message that pops up. Is this correct?